frameworks/H2OAutoML/exec.py

```diff
@@ -36,7 +36,11 @@ def run(dataset: Dataset, config: TaskConfig):
         nthreads = config.framework_params.get('_nthreads', config.cores)
 
         log.info("Starting H2O cluster with %s cores, %sMB memory.", nthreads, config.max_mem_size_mb)
+        max_port_range = 49151
+        min_port_range = 1000
```

Comment:
I guess you wanted to limit to registered ports
```suggestion
        min_port_range = 1024
```